src/types.ts

```diff
@@ -82,6 +82,16 @@ export interface ICourierProfilePostParameters {
   recipientId: string;
   profile: object;
 }
+// POST /profiles/{id}/lists
+export type List = Omit<ICourierList, "id"> & {
```

Comment:
in all our create/update endpoints, we refer to list id by `listId` and not `id` but rest of the interface is same.